connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -364,7 +403,7 @@ public void testCreateConnectorNoName() throws Throwable {
 
         PowerMock.replayAll();
 
-        connectorsResource.createConnector(FORWARD, bodyIn);
+        connectorsResource.createConnector(FORWARD, null, bodyIn);
```

Comment:
Maybe instead of `null` we could use `NO_HEADERS` instead?

Reply:
I'm not sure about this. Would you like to have HttpHeaders nullHeaders = null;

Reply:
I would like to see:
```
connectorsResource.createConnector(FORWARD, NULL_HEADERS, bodyIn);
```
to make it much more clear what the second parameter is and that most of the tests are not specifying headers.

Reply:
Done. Thanks for clearer instruction.